src/apps/api/views/competitions.py

```diff
@@ -80,6 +91,23 @@ def toggle_publish(self, request, pk):
         competition.save()
         return Response({"published": competition.published})
 
+    @action(detail=True, methods=('POST',))
+    def register(self, request, pk):
+        competition = self.get_object()
+        user = request.user
+        participant = CompetitionParticipant.objects.create(competition=competition, user=user)
+        if user in [competition.created_by] + list(competition.collaborators.all()):
```

Comment:
> `[competition.created_by] + list(competition.collaborators.all()):`

we do this pattern _a lot_ it feels like, should we have `competition.all_organizers` property, or something? Hmm...